Packet++/header/TcpReassembly.h

```diff
@@ -102,13 +102,13 @@ struct ConnectionData
 	 * Set startTime of Connection
 	 * @param[in] startTime integer value
 	 */
-	void setStartTime(const timeval &startTime) { this->startTime = startTime; }
+	void setStartTime(const timeval &_startTime) { startTime = _startTime; }
 
 	/**
 	 * Set endTime of Connection
 	 * @param[in] endTime integer value
 	 */
-	void setEndTime(const timeval &endTime) { this->endTime = endTime; }
+	void setEndTime(const timeval &_endTime) { endTime = _endTime; }
```

Comment:
These changes break doxygen: https://cirrus-ci.com/task/5505579729289216

I'd call the variables `startTimeValue` instead of `_startTime` and `endTimeValue` instead of `_endTime`.

To satisfy doxygen we should also change the variable names in the methods' documentation